backend/capellacollab/sessions/hooks/jupyter.py

```diff
@@ -77,3 +91,60 @@ def pre_session_termination_hook(
 
     def _determine_base_url(self, username: str):
         return f"{self._jupyter_public_uri.path}/{username}"
+
+    def _get_project_share_volume_mounts(
+        self,
+        db: orm.Session,
+        token: dict[str, t.Any],
+        tool: tools_models.DatabaseTool,
+    ) -> list[operators_models.PersistentVolume]:
+        volumes = []
+
+        accessible_models_with_workspace_configuration = [
+            model
+            for model in toolmodels_crud.get_models_by_tool(db, tool.id)
+            if model.configuration
+            and "workspace" in model.configuration
+            and self._is_project_member(model, token, db)
+        ]
+
+        for model in accessible_models_with_workspace_configuration:
+            assert model.configuration
```

Comment:
Do we need this here for mypy/pylint? Because as far as I can see `accessible_models_with_workspace_configuration` should only contain models where `model.configuration` exists

Reply:
Yes, mypy doesn't get the list comprehension. It did work with a simple for loop, but list comprehension seems to be to complex for mypy.